app/restaurant-owner/[restaurantId]/page.jsx

```
@@ -43,11 +43,13 @@ const FoodSalesPage = () => {
           <p className="text-3xl font-bold text-secondary-orange tracking-wider py-2  ">
             ₹{" "}
             {isSuccess
-              ? data.data.foodSales.reduce(
-                  (accumulator, currentValue) =>
-                    accumulator + parseFloat(currentValue.sum),
-                  0
-                )
+              ? data.data.foodSales
+                  .reduce(
+                    (accumulator, currentValue) =>
+                      accumulator + parseFloat(currentValue.sum),
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)